kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -221,6 +224,18 @@ public static ConfigDef newConfigDef() {
           new SseAlgorithmRecommender()
       );
 
+      configDef.define(
+          SSE_KEY,
+          Type.STRING,
+          SSE_KEY_DEFAULT,
+          Importance.LOW,
+          "The S3 Server Side Encryption Customer Key.",
```

Comment:
I'd recommend sticking as closely as possible to Amazon's terminology here. I'd refer to this as "customer-provided server-side encryption key (SSE-C)"

Reply:
Thanks, will fix

Reply:
I checked the docs and it looks like all of these strings parrot the HTTP header strings: https://docs.aws.amazon.com/AmazonS3/latest/dev/ServerSideEncryptionCustomerKeys.html. I could change them all to be customer (document) friendly, what do you think @kkonstantine?